working/augmentation-libraries/feature-specification.md

```diff
@@ -142,59 +141,58 @@ It is a compile-time error if:
-import augment 'some_augmentation.dart';
+augment 'some_augmentation.dart';
 ```
 
 This directive tells the compiler to read the given library augmentation and
 merge its declarations into this library. It is a compile-time error if:
 
-*   The library referenced in an `import augment` directive is not an
-    augmentation and does not have a `library augment` directive.
+*   The URI referenced in an `augment` directive is not an augmentation and does
+    not have a `augment library` directive.
 
-*   The library referenced in a `library augment` directive does not have an
-    `import augment` directive pointing back to this augmentation.
+*   The library referenced in a `augment library` directive does not have an
+    `augment` directive pointing back to this augmentation.
 
-*   The same augmentation library is applied more than once. *In other words,
-    you can't have redundant `import augment` directives that point to the same
-    library.*
+*   The same augmentation is applied more than once. *In other words, you can't
+    have redundant `augment` directives that point to the same library.*
 
 *   The main library and its augmentations do not all have the same language
     version. There is only one user-visible library at the end, and it should
     have a consistent version across its entire surface area. *An augmentation
-    library does not automatically inherit any language version from the main
-    library and may need an explicit language version comment of its own in
-    order to adhere to this requirement.*
+    does not automatically inherit any language version from the main library
+    and may need an explicit language version comment of its own in order to
+    adhere to this requirement.*
 
-Since the main library and its augmentation both point to each other, these
+Since the main library and its augmentations both point to each other, these
 rules imply that a given augmentation file can only be used to augment a single
 library.
 
 ### Merge order
 
 A library may apply multiple augmentations to itself. Also, augmentation files
-may themselves contain `import augment` directives. The entire tree of
-augmentations is recursively applied to the main library. The merge order is
-defined as a depth-first pre-order traversal of the `import augment` directives
-starting at the main library.
+may themselves contain `augment` directives. The entire tree of augmentations is
+recursively applied to the main library. The merge order is defined as a
+depth-first pre-order traversal of the `augment` directives starting at the main
+library
 
-Within a single augmentation library, you may augment the same declaration
-multiple times, whether they are top level or nested declarations. The merge
-order of these is defined as the source order of the augmentations.
+Within a single augmentation file, you may augment the same declaration multiple
+times, whether they are top level or nested declarations. The merge order of
```

Comment:
Nit: "they" should refer to a plural object, but I think it refers to "same declaration", so "it is" (or better "the declaration is", unqualified "it"s risk being ambiguous.)